### PR TITLE
squashed simplified store

### PR DESCRIPTION
**Pull request by collin**
@kand @daytonn 

* removes the `Field` class
* removes the `Action` class
* replaces `Store` subclasses with POJO field schema

### CODE REVIEW

**Comment by daytonn** [comment]
👍

File: .eslintrc [hunk 1]
```diff
@@ -196,7 +198,6 @@
     "space-before-function-paren": 0,
     "space-in-parens": [1, "never"],
     "space-infix-ops": 1,
-    "space-return-throw-case": 1,
```
Comment (collin) [inline .eslintrc]: Actually, upon looking up this rule I don't know why I was breaking it, going to look into it:
http://eslint.org/docs/rules/space-return-throw-case

Comment (collin) [inline .eslintrc]: Ah, that's right:

`Replacement notice: This rule was removed in ESLint v2.0 and replaced by keyword-spacing rule.`

These rules were removed.

**Comment by collin** [comment]
Merging this for @daytonn to pull into his campaign-display tests.

**Pull request merged**